MediaRecorder.html

```diff
@@ -712,12 +713,19 @@
         <h2>Attributes</h2>
         <dl data-link-for="BlobEvent" data-dfn-for="BlobEvent" class=
         "attributes">
-          <dt><dfn><code>data</code></dfn> of type <span class=
-          "idlAttrType"><a>Blob</a></span>, readonly</dt>
-          <dd>Returns a <a
-          href="https://www.w3.org/TR/FileAPI/#blob"><code>Blob</code></a>
-          object whose type attribute indicates the encoding of the blob
-          data.</dd>
+          <dt><dfn><code>data</code></dfn> of type <span class="idlAttrType">
+          <a href="https://www.w3.org/TR/FileAPI/#blob">Blob</a></span>,
+          readonly</dt>
+          <dd>The encoded blob. Its
+          <a href="https://www.w3.org/TR/FileAPI/#dfn-type">
+          type</a> attribute correlates to the selected
+          <a href="#dom-mediarecorder-mimeType">mimeType</a>.</dd>
+
+          <dt><dfn><code>timecode</code></dfn> of type <span class "idlAttrType">
+          <a href="https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">
+          DOMHighResTimeStamp</a></span>, readonly</dt>
+          <dd><code>Timecode</code> associated with the Blob data, relative to
+          the first produced <code>blob</code>.</dd>
```

Comment:
> What does timecode mean in that case?

If the recorded blob contains a lot of stuff that could have independent `timecode`s, I'd say the `BlobEvent` `timecode` should be the earliest one.

> Why `DomHighResTimestamp`? 

The alternative [`DOMTimestamp`](https://www.w3.org/TR/DOM-Level-3-Core/core.html#DOMTimeStamp) is just a number of milliseconds in a `long long` format, whereas [`DomHighResTimestamp`](https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp) conveys the meaning of time measured from a given origin (and, is `double`, which is good).

I think we should specify that the `origin` in this case is the first `Blob` produced by MR (but we don't have to specify this value to be zero).

`timecode` comes from the film industry: https://en.wikipedia.org/wiki/Timecode